pages/index.tsx

```diff
@@ -207,6 +207,47 @@ serve((req) => new Response("Hello World\\n"), { port: 8000 });`;
             <a class="link" href="/x">deno.land/x</a>.
           </p>
         </div>
+        <div class="max-w-screen-sm mx-auto px-4 sm:px-6 md:px-8 mt-20">
+          <a class="hover:underline" href="#examples">
+            <h3 class="font-bold text-xl" id="examples">
+              Examples
+            </h3>
+          </a>
+          <p class="my-4 text-gray-700">
+            Here are some examples that you can use to get started immediately.
+          </p>
+          <ol class="ml-8 list-disc text-gray-700">
+            <li>
+              <a href="https://examples.deno.land/hello-world/" class="link">
```

Comment:
Remove the trailing slash in the URL. With the trailing slash it 404's (I'll fix this upstream).